### PR TITLE
test: emergency patch to repair the Windows CI

### PR DESCRIPTION
**Pull request by compnerd**
The removal of a key not in the set is an error.

Replace this paragraph with a description of your changes and rationale. Provide links to external references/discussions if appropriate.

Resolves [SR-NNNN](https://bugs.swift.org/browse/SR-NNNN).

### CODE REVIEW

**Comment by compnerd** [comment]
CC: @yln

**Comment by compnerd** [comment]
@swift-ci please smoke test and merge

**Comment by yln** [comment]
Thanks for cleaning up after me. Did not realize that remove throws if value is not present.

File: test/lit.cfg [hunk 1]
```diff
@@ -1500,7 +1500,8 @@ else:
         config.available_features.add('libdispatch')
     else:
         # TSan runtime requires libdispatch on non-Apple platforms
-        config.available_features.remove('tsan_runtime')
+        if 'tsan_runtime' in config.available_features:
```
Comment (yln) [inline test/lit.cfg]: We can use `discard` instead of `remove` (does not throw if not present).

Comment (compnerd) [reply test/lit.cfg]: Good idea, I'll update this.

**Comment by compnerd** [comment]
@swift-ci please smoke test and merge

**Comment by compnerd** [comment]
@swift-ci please smoke test and merge

**Comment by compnerd** [comment]
@swift-ci please smoke test Linux platform

**Comment by compnerd** [comment]
@swift-ci please smoke test Linux platform

**Pull request merged**